Assets/Scripts/Characters/EntityStats.cs

```diff
@@ -2,7 +2,7 @@
 using System.Collections.Generic;
 using UnityEngine;
 
-public class EntityStats : MonoBehaviour {
+public class EntityStats {
```

Comment:
This was causing a warning to be logged as Unity dislikes MonoBehavior instances being created with 'new'.